test/Sema/package_resilience_access.swift

```diff
@@ -63,141 +286,263 @@ package func f(_ arg: PublicStruct) -> Int {
 
 package func g(_ arg: PkgStruct) -> Int {
   return arg.data
 }
 
-// CHECK: // g(_:)
-// CHECK-NEXT: sil @$s6Client1gySi5Utils9PkgStructVF : $@convention(thin) (PkgStruct) -> Int {
-// CHECK-NEXT: // %0 "arg"                                       // users: %2, %1
-// CHECK-NEXT: bb0(%0 : $PkgStruct):
-// CHECK-NEXT:   debug_value %0 : $PkgStruct, let, name "arg", argno 1 // id: %1
-// CHECK-NEXT:   %2 = struct_extract %0 : $PkgStruct, #PkgStruct.data // user: %3
-// CHECK-NEXT:   return %2 : $Int                                // id: %3
-// CHECK-NEXT: } // end sil function '$s6Client1gySi5Utils9PkgStructVF'
+// CLIENT-RES-LABEL: // g(_:)
+// CLIENT-RES-NEXT: sil [ossa] @$s6Client1gySi5Utils9PkgStructVF : $@convention(thin) (@in_guaranteed PkgStruct) -> Int
+// CLIENT-RES-LABEL: // PkgStruct.data.getter
+// CLIENT-RES-NEXT: sil @$s5Utils9PkgStructV4dataSivg : $@convention(method) (@in_guaranteed PkgStruct) -> Int
+
+// CLIENT-NONRES-LABEL: // g(_:)
+// CLIENT-NONRES-NEXT: sil [ossa] @$s6Client1gySi5Utils9PkgStructVF : $@convention(thin) (PkgStruct) -> Int
+
+package func gx(_ arg: UfiPkgStruct) -> Int {
+  return arg.data
+}
+
+// CLIENT-RES-LABEL: // gx(_:)
+// CLIENT-RES-NEXT: sil [ossa] @$s6Client2gxySi5Utils12UfiPkgStructVF : $@convention(thin) (@in_guaranteed UfiPkgStruct) -> Int
+// CLIENT-RES-LABEL: // UfiPkgStruct.data.getter
+// CLIENT-RES-NEXT: sil @$s5Utils12UfiPkgStructV4dataSivg : $@convention(method) (@in_guaranteed UfiPkgStruct) -> Int
+
+// CLIENT-NONRES-LABEL: // gx(_:)
+// CLIENT-NONRES-NEXT: sil [ossa] @$s6Client2gxySi5Utils12UfiPkgStructVF : $@convention(thin) (UfiPkgStruct) -> Int
 
 package func m<T>(_ arg: PkgStructGeneric<T>) -> T {
   return arg.data
 }
 
-// CHECK: // m<A>(_:)
-// CHECK-NEXT: sil @$s6Client1myx5Utils16PkgStructGenericVyxGlF : $@convention(thin) <T> (@in_guaranteed PkgStructGeneric<T>) -> @out T {
-// CHECK-NEXT: // %0 "$return_value"                             // user: %4
-// CHECK-NEXT: // %1 "arg"                                       // users: %3, %2
-// CHECK-NEXT: bb0(%0 : $*T, %1 : $*PkgStructGeneric<T>):
-// CHECK-NEXT:   debug_value %1 : $*PkgStructGeneric<T>, let, name "arg", argno 1, expr op_deref // id: %2
-// CHECK-NEXT:   %3 = struct_element_addr %1 : $*PkgStructGeneric<T>, #PkgStructGeneric.data // user: %4
-// CHECK-NEXT:   copy_addr %3 to [init] %0 : $*T                 // id: %4
-// CHECK-NEXT:   %5 = tuple ()                                   // user: %6
-// CHECK-NEXT:   return %5 : $()                                 // id: %6
-// CHECK-NEXT: } // end sil function '$s6Client1myx5Utils16PkgStructGenericVyxGlF'
+// CLIENT-RES-LABEL: // m<A>(_:)
+// CLIENT-RES-NEXT: sil [ossa] @$s6Client1myx5Utils16PkgStructGenericVyxGlF : $@convention(thin) <T> (@in_guaranteed PkgStructGeneric<T>) -> @out T {
+// CLIENT-RES-NEXT: // %0 "$return_value"                             // user: %6
+// CLIENT-RES-NEXT: // %1 "arg"                                       // users: %4, %2
```

Comment:
Are these `// users:` comment necessary to the test? John recently warned against superfluously strict SIL tests and pointed out most of these can usually be removed if they're not what's actually being tested.

Reply:
Not necessary. I'll remove these.